### PR TITLE
{bio}[intel/2017b] Oases v0.2.08 -kmer_101

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/8cf404d1012d74602107659f1e5632b7 for a full test report.

File: easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+description = """Oases is a de novo transcriptome assembler designed to produce transcripts from
+ short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+velvetver = '1.2.10'
+
+source_urls = [
+    'http://www.ebi.ac.uk/~zerbino/%(namelower)s',
+    'http://www.ebi.ac.uk/~zerbino/velvet',
```
Comment (verdurin) [inline easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb]: @smoors  These source URLs don't work anymore. Is the code hosted elsewhere now?

Comment (smoors) [reply easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb]: @verdurin For me they still work, could you please try again?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2110.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b68ed072dda425cd37129c58a055e28b for a full test report.

File: easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb]: Still works for me too, still provides exact same source tarballs (SHA256 checksum is identical for fresh downloads).

So +1 on trying again @verdurin ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2652.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/92f73ae756cc8e4e7d20b4798af65df3 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 8 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/654a2336ac96f12de1d844fd7c0e8dbf for a full test report.

**Comment by verdurin** [comment]
Ignore that failed build.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/5c3c8a79e23e1304146ea296ed8eda9b for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @smoors!

**Pull request merged**